project.clj

```diff
@@ -8,32 +12,77 @@
 
   :pedantic? :abort
 
-  :dependencies [[org.jruby/jruby-core "1.7.26"
-                  :exclusions  [com.github.jnr/jffi
-                                com.github.jnr/jnr-x86asm
-                                joda-time
-                                org.yaml/snakeyaml]]
+  ;; Note that there is a fair amount of duplication between the
```

Comment:
Might be able to do that.  I thought about it but hadn't played around it with it.  I'll do that and, if I can get it to work, see if we think that approach would look better.  Thanks!